src/VisualizationTests/CameraTests.cs

```diff
@@ -48,11 +48,11 @@ public void Camera_GoodSaveData_LoadsCorrectly()
 
             var cam = ((HelixWatch3DViewModel)ViewModel.BackgroundPreviewViewModel).Camera;
 
-            //<Cameras>
+            //<Camera>
```

Comment:
@mjkkirschner I am not sure the best way to maintain the old tests since the variable names have been updated.  The test below checks the same data using the refactored naming convention.  The tests won't compile if I leave them as is because the old variables no longer exist.  I guess my question is how do I maintain full support for XML and testing legacy graphs but still follow the subtask which was to "Rename Cameras to Camera in both AVP and Dynamo".  It seems like the only way to do this is to only rename during serialization and deserialization so the JSON contains `Camera` for CoGS but AVP and Dynamo still use `Cameras`.  Is there a better approach to doing this?